extensions/wikia/Chat2/Chat_setup.php

```diff
@@ -51,6 +51,8 @@
 $wgHooks['LogLine'][] = 'ChatHooks::onLogLine';
 $wgHooks['UserGetRights'][] = 'ChatHooks::onUserGetRights';
 $wgHooks['ParserFirstCallInit'][] = 'ChatWidget::onParserFirstCallInit';
+// needs to be prepended so that it can abort the hook before the other handlers run
+$wgHooks['BeforePageDisplay'] = [ 'ChatHooks::abortBeforePageDisplayOnSpecialChat' ] + $wgHooks['BeforePageDisplay'];
 
 // logs
 $wgLogTypes[] = 'chatban';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "chatban" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)